.env.example

```diff
@@ -2,9 +2,17 @@ APP_NAME=Lychee
 APP_ENV=production
 APP_KEY=
 APP_DEBUG=false
+# This MUST contain the host name up to the Top Level Domain (tld) e.g. .com, .org etc.
 APP_URL=http://localhost
 APP_FORCE_HTTPS=false
 
+# If using Lychee in a sub folder, specify the path after the tld here.
+# For example for https://lychee.test/personal-gallery/public
+# Set APP_URL=https://lychee.test
+# and APP_DIR=/personal-gallery/public
```

Comment:
```suggestion
# For example for https://lychee.test/path/to/lychee
# Set APP_URL=https://lychee.test
# and APP_DIR=/path/to/lychee
```
I don't like including `public` there. To me it's marginally more likely to result in exposed roots.